pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -1084,6 +1085,8 @@ func (p *Pod) FindMatchingWorkloads(ctx context.Context, c client.Client, r reco
 	activePods := p.runnableOrSucceededPods()
 	inactivePods := p.notRunnableNorSucceededPods()
 
+	log.Info("pods", "active", activePods, "inactive", inactivePods)
```

Comment:
```suggestion
	log.V(5).Info("pods", "active", activePods, "inactive", inactivePods)
```
These are not just pod names, but entire pods (up to 1.5Mi each. potentially), so logging them unconditionally is excessive. 

Maybe we also wrap this with the level check for the logger.

Reply:
I'll just remove it, It was just for debug.